Aliases/map - move - over/5eb8d24b-d660-42e5-bbea-47a659bb0827 - Main GVAR.gvar

```diff
@@ -177,172 +198,7 @@ if c:
-  #If there is a -object call
-  if args.last('object'):
-   if args.last('object') in ('none', 'None', True, 'true', 'True'):
-    objects = []
-    desc.append(f"Objects reset")
-   else:
-    foundObject = ""
-    argsObject = args.last("object").lower().split(":")
-    for object in presetObjects:
-     if argsObject[1] in object.lower():
-      foundObject = presetObjects[object]
-      break
-    if foundObject:
-     objectLocation = argsObject[0]
-     loadedObject=objectLocation+foundObject
-     objects.append(loadedObject)
-     desc.append(f"Object added `{object}:{loadedObject}`")
-    else:
-     desc.append("Could not find object")
-  if args.last('deleteobject'):   
-   foundObject = "There is no object by that name"
-   argsObject = args.last("deleteobject").lower().split(":")
-   for object in presetObjects:
-    if argsObject[1] in object.lower():
-     foundObject = presetObjects[object]
-     break
-   objectLocation = argsObject[0]
-   loadedObject=objectLocation+foundObject
-   objects.remove(loadedObject)
-   desc.append(f"Object `{object}:{loadedObject}` was removed") 
-  # If there is a -mapattach, and its a valid target in init
-  if args.last('mapattach', args.last('attach')) and gt(args.last('mapattach', args.last('attach'))):
-   # First check if there is an existing mapattach, and if its the same as the new one, then remove their map effect
-   if typeof(mapattach)=="SimpleCombatant" and gt(args.last('mapattach', args.last('attach'))).name != mapattach.name:
-    mapattach.remove_effect('map')
-   # Then, set the new mapattach
-   mapattach = gt(args.last('mapattach', args.last('attach')))
-  # If theres mapattach set, and they currently have a map effect, remove it
-  if mapattach and mapattach.get_effect('map'):
-   mapattach.remove_effect('map')
-  # Format everything appropriately
-  setMapView = f"{mapviewlocation}:{mapviewsize}"
-  neweffect = f"""{f"Size: {mapsize}" if mapsize else ""}{f" ~ Background: {mapbg}" if mapbg else ""}{f" ~ Options: {mapoptions}" if mapoptions else ""}{f" ~ Walls: {', '.join(walls)}" if walls else""}{f' ~ Objects: {"/".join(objects)}' if objects else ""}{f" ~ View: {setMapView}" if setMapView.strip(':') else ''}""".strip(" ~")
-  # If mapattach exists, apply the new effect, and display
-  if mapattach:
-   mapattach.add_effect('map', f"""-attack "||{neweffect}" """)
-   desc.append(f"Map settings attached to `{mapattach.name}`")
-  # Otherwise, show that it was changed, but not saved
-  else:
-   # Look for a combatant named DM or Map, auto attach
-   for maptarg in ("map", "dm", "lair"):
-     if c.get_combatant(maptarg):
-       mapattach = c.get_combatant(maptarg)
-       mapattach.add_effect('map', f"""-attack "||{neweffect}" """)
-       desc.append(f"""Autoattached map settings to {mapattach.name}""")
-       break
-   else:
-    desc.append("Map settings changed, but no map attach target was found. Settings not saved.")
-</drac2>
+
```

Comment:
Moved this code block lower so when referencing "out" for the combatants info it can